cime_config/machines/cmake_macros/nvidiagpu_pm-gpu.cmake

```diff
@@ -6,8 +6,8 @@ if (COMP_NAME STREQUAL gptl)
 endif()
 string(APPEND CPPDEFS " -DTHRUST_IGNORE_CUB_VERSION_CHECK")
 string(APPEND CMAKE_CUDA_FLAGS " -ccbin CC -O2 -arch sm_80 --use_fast_math")
-string(APPEND CMAKE_EXE_LINKER_FLAGS " -acc -gpu=cc70,cc60 -Minfo=accel")
+string(APPEND CMAKE_EXE_LINKER_FLAGS " -acc -gpu=cc80 -Minfo=accel")
 set(SCC "cc")
 set(SCXX "CC")
 set(SFC "ftn")
-string(APPEND CMAKE_Fortran_FLAGS " -acc -gpu=cc70,cc60 -Minfo=accel")
+string(APPEND CMAKE_Fortran_FLAGS " -acc -gpu=cc80 -Minfo=accel")
```

Comment:
As we discussed in our Slack huddle, let's make this its own commit and submit it to E3SM as a bug fix.

Reply:
I created a new commit only for this change. Thanks!

Reply:
@grnydawn, can you make a PR to E3SM as well with this commit?

Reply:
Sure, I can help with that.  Your fork of Omega (https://github.com/grnydawn/Omega) is also a fork of https://github.com/E3SM-Project/E3SM.  You can add it as a remote to wherever you usually develop E3SM:
```
git remote add grnydawn/Omega git@github.com:grnydawn/Omega.git
git fetch grnydawn/Omega -p
```
Then, make a new branch from E3SM master however you would normally do that, e.g.:
```
git fetch origin -p
git reset --hard origin/master
git checkout -b ykim/machinefiles/fix-nvidiagpu-pm-gpu
```
They, you just need to cherry-pick the commit from here to the branch:
```
git cherry-pick 294a5ac4d453c61999dee1e170597d80f73822ed
```
where I got the hash of the commit from the copy icon next to the commit listed here:
https://github.com/E3SM-Project/Omega/pull/85/commits

Reply:
From there, just push the branch to `origin` as usual and make a PR to E3SM as usual.

Reply:
Ideally, we will revert the commit here on Omega before merging `master` into Omega's `develop` so it's clear that the E3SM version of the commit, rather than this one, is considered to be the "right" one.

Reply:
Thanks a lot @xylar for detail instructions. Before proceeding to create a PR, I have one question on the changes in Subproject commits. From running git diff HEAD~1 at "ykim/machinefiles/fix-nvidiagpu-pm-gpu" I got following Subproject commit changes. I wonder if I could ignore the git log messages:

```
diff --git a/components/elm/src/external_models/fates b/components/elm/src/external_models/fates
index 42d804ba54..698a8df848 160000
--- a/components/elm/src/external_models/fates
+++ b/components/elm/src/external_models/fates
@@ -1 +1 @@
-Subproject commit 42d804ba54d0cf013a9737018ff9920e0c9808ea
+Subproject commit 698a8df848ecdb81aa72fee6c86be2c41b2545e9
diff --git a/externals/ekat b/externals/ekat
index fb4babcf3a..2ff5853316 160000
--- a/externals/ekat
+++ b/externals/ekat
@@ -1 +1 @@
-Subproject commit fb4babcf3a24c14e9e94d71df152a7f097805c14
+Subproject commit 2ff5853316e15d4e8004c21890329fd257fa7459
diff --git a/externals/haero b/externals/haero
index 77c4a45584..574787f9b3 160000
--- a/externals/haero
+++ b/externals/haero
@@ -1 +1 @@
-Subproject commit 77c4a45584f2bd3bf38a18ea5bc24ef9ce3776a2
+Subproject commit 574787f9b31cc77b7902f07e73e10dcd18cdf29d
diff --git a/externals/mam4xx b/externals/mam4xx
index a19bc75891..8f6af492c8 160000
--- a/externals/mam4xx
+++ b/externals/mam4xx
@@ -1 +1 @@
-Subproject commit a19bc75891e235071426840742a2c530bbea9e52
+Subproject commit 8f6af492c8627ebe5ec7904c3185ab8294e49a12
diff --git a/externals/scorpio b/externals/scorpio
index cdd541e0cd..de0b1ca220 160000
--- a/externals/scorpio
+++ b/externals/scorpio
@@ -1 +1 @@
-Subproject commit cdd541e0cd708bece13b1f3ee42f66dfd6440aa7
+Subproject commit de0b1ca2200f62c6eb5e3fd40147965409e97123
```

Reply:
I think you can probably ignore those.  It will be clear as soon as you make a PR whether your commits contain any changes to these submodules.  I think the more likely thing is simply that they have been changed between different branches you checked out and you would just need to do a new recursive update to make your submodules match what they are supposed to be on the current branch.

Reply:
Thanks, @xylar. I think I will check if my commit has older submodules because I have not updated submodules recently, and then proceed to create a PR.